lib/brainstem/api_docs/controller.rb

```diff
@@ -101,13 +103,19 @@ def tag_groups
       #
       def contextual_documentation(key)
         default_configuration.has_key?(key) &&
-          !default_configuration[key][:nodoc] &&
+          !nodoc_for?(default_configuration[key]) &&
           default_configuration[key][:info]
       end
 
       def valid_sorted_endpoints
         endpoints.sorted_with_actions_in_controller(const)
       end
+
+      private
+
+      def nodoc_for?(config)
+        !!(config[:nodoc] || (config[:internal] && !internal))
```

Comment:
This is great! Now I can use our internal docs as a source of truth instead of just opting for source code. 👍